hvpy/io.py

```diff
@@ -1,11 +1,12 @@
+import os
 from enum import Enum, auto
-from typing import Any, Dict
+from typing import Any, Dict, Optional
 
 from pydantic import BaseModel
 
-__all__ = ["HvpyParameters", "OutputType"]
+__all__ = ["HvpyParameters", "OutputType", "set_api"]
 
-BASE_URL = "https://api.helioviewer.org/v2/"
```

Comment:
On the surface, I don't see why you can't just set the value here if the OS environmental variable exists?